src/integ-test/bdd-test/features/primitives/paGetPaymentRes_syntax_ko.feature

```diff
@@ -0,0 +1,288 @@
+      | entityUniqueIdentifierType  | None                                                                                                                                                                                                                                                              | SIN_PGPR_50   |
+      | entityUniqueIdentifierType  | Empty                                                                                                                                                                                                                                                             | SIN_PGPR_51   |
+      | entityUniqueIdentifierType  | H                                                                                                                                                                                                                                                                 | SIN_PGPR_52   |
+      | entityUniqueIdentifierValue | None                                                                                                                                                                                                                                                              | SIN_PGPR_53   |
+      | entityUniqueIdentifierValue | Empty                                                                                                                                                                                                                                                             | SIN_PGPR_54   |
+      | entityUniqueIdentifierValue | lunghezza >16char                                                                                                                                                                                                                                                 | SIN_PGPR_55   |
+      | fullName                    | None                                                                                                                                                                                                                                                              | SIN_PGPR_56   |
+      | fullName                    | Empty                                                                                                                                                                                                                                                             | SIN_PGPR_57   |
+      | fullName                    | lunghezza >16char                                                                                                                                                                                                                                                 | SIN_PGPR_58   |
```

Comment:
this test fails

Reply:
Insert string "test di prova per una lunghezza superiore a 70 caratteri alfanumerici ," in line 234